.github/workflows/python-package.yaml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    - main
+  pull_request:
+    - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
```

Comment:
i don't think 39 is standard in ubuntu so maybe we can skip it to save time?

Reply:
Ah sure